src/components/CreateVmWizard/steps/BasicSettings.js

```diff
@@ -270,11 +271,26 @@ class BasicSettings extends React.Component {
         changes.initSshKeys = template.getIn(['cloudInit', 'sshAuthorizedKeys'])
         changes.initTimezone = template.getIn(['cloudInit', 'timezone'])
         changes.initCustomScript = template.getIn(['cloudInit', 'customScript'])
+
+        if (changes.initTimezone && isOsWindows(changes.operatingSystemId, this.props.operatingSystems)) {
+          changes.enableSysPrepTimezone = true // Configure Timezone checkbox should be checked if sysprep timezone is set by a chosen template
+          changes.selectedSysprepTimezone = changes.initTimezone // select the template based sysprep timezone in the appropriate drop down
+        } else {
+          changes.enableSysPrepTimezone = false
+          changes.initTimezone = '' // reset the sysprep timezone value
+          changes.selectedSysprepTimezone = timezones.find(timezone => timezone.value.startsWith('(GMT) Greenwich')).id // display the the default GMT sysprep timezone as selected in the appropriate drop down, same as in Admin Portal
+        }
+
         break
 
       case 'operatingSystemId':
         changes[field] = value
         changes.timeZone = this.checkTimeZone(value, this.props.data.templateId)
+
+        // when Windows os selected, set the sysprep timezone to the last selected sysprep timezone if sysprep already enabled together with Configure Timezone checkbox checked
```

Comment:
Bug scenario:
1. set sysprep timezone for Windows 7 to non-deafult value i.e. Dateline Standard Time
2. uncheck Configure Timezone checkbox
3. change OS to Windows 8 (Dateline TZ is disabled but visible)
4. go to Summary

Expected result: sysprep section contains Datelin TZ as visible on the screen
Actual result: no data

Reply:
This scenario isn't buggy, the actual result is also the expected one: no sysprep timezone is displayed in the _Review_ step, if sysprep timezone was not configured (because _Configure Timezone_ checkbox unchecked in the step 2) and only `Sysprep enabled` is displayed because the sysprep remained enabled, obviously (step 1).

Reply:
Let's bring the facts here:
1. sysprep is enabled
2. in sysprep section user sees a (non-default) value in time zone box. 
3. the contract on Basic step seems to be `visible == selected`. Disabled is still visible.
4. the visible value is not present on the summary step

It's not obvious that clicking the Configure Timezone checkbox is required to select the time zone. I was convinced this is needed only to change the time zone. If this is by design then I would change the design. Perhaps it makes sense to simple hide time zone select if Configure Timezone is not checked? 
Actually this scenario feels wrong to me even if user sees there the default time zone (although it's then less confusing).

Reply:
> 4. the visible value is not present on the summary step

That's correct. Displaying this value in the _Review_ step would lead to the feeling that sysprep TZ is configured even when it is not. This was always working like that: the sysprep TZ was visible in the Review step only if it was configured. And in the Basic settings, the last selected sysprep TZ is visible but grey, disabled, together with the checkbox unchecked. I think that's enough understandable for a normal user. It's in webadmin like that, too. After successful creating the VM, you will see the correct sysprep TZ in the VM Details.

> Perhaps it makes sense to simple hide time zone select if Configure Timezone is not checked?

With such a change, we would make it inconsistent with the webadmin. The main purpose of this PR is the opposite.

Reply:
> we would make it inconsistent with the webadmin

That's true. To my surprise Web Admin behaves as you described. 
Let's call it a feature then:)